src/core/display/Container.js

```diff
@@ -606,17 +606,19 @@ Container.prototype.renderCanvas = function (renderer)
 
 /**
  * Destroys the container
- * @param [destroyChildren=false] {boolean} if set to true, all the children will have their destroy method called as well
+ * @param [destroyChildren=false] {boolean} Should it destroy any children?
+ * @param [destroyTexture=false] {boolean} Should it destroy the current texture of any children?
+ * @param [destroyBaseTexture=false] {boolean} Should it destroy the base texture?
  */
-Container.prototype.destroy = function (destroyChildren)
+Container.prototype.destroy = function (destroyChildren, destroyTexture, destroyBaseTexture)
```

Comment:
While this works, it makes me uncomfortable because it makes container be knowledgable about how child objects are setup.

Instead using an object as the param means you can put any keys int he object you want and only the children need to be knowledgeable about them. See this comment: https://github.com/pixijs/pixi.js/pull/2536#issuecomment-215141153